app.py

```diff
@@ -188,6 +188,7 @@ class MQStatus(object):
 
         hash_ = data['hash']
         result = data['validation']
+        service = data.get('service')
```

Comment:
used data.get here because some services will likely not be sending this particular field until they update their code.

Reply:
Shouldn’t we update the IPP that this field is going to be mandatory?

Reply:
@Glutexo I've added the suggestion to the IPP, along with the other one you had about regex

:+1:

Reply:
As for the regex, there might be some will to unify this with IPP 12 – REST api object names. See my new comment in IPP 1.

Reply:
@Glutexo This was discussed in the IPWG this morning and they didn't feel that there was any pressure to have them match necessarily.

Reply:
Aside from original discussion about IPP:

This won’t work, because you’re [sending](https://github.com/RedHatInsights/insights-upload/blob/8e2d19bb830ff6a09735c167366d7cf89dfb6d1a/docker/consumer/app.py#L60) the field under the _validating_service_ key instead of _service_. This might not be an issue depending on how you (or the IPP staff) decide on naming, but in the current state of the code it’s a bug.